src/plugins/ini/ini.c

```diff
@@ -198,13 +357,11 @@ int elektraIniOpen(Plugin *handle, Key *parentKey ELEKTRA_UNUSED)
 	KeySet *config = elektraPluginGetConfig (handle);
 	IniPluginConfig *pluginConfig = (IniPluginConfig *)elektraMalloc (sizeof (IniPluginConfig));
 	Key *multilineKey = ksLookupByName (config, "/multiline", KDB_O_NONE);
-	Key *autoSectionKey = ksLookupByName(config, "/autosections", KDB_O_NONE);
-	Key *toMetaKey = ksLookupByName(config, "/meta", KDB_O_NONE);
-	Key *preserveKey = ksLookupByName(config, "/preserveorder", KDB_O_NONE);
-	pluginConfig->preserverOrder = preserveKey != 0;	
+	Key *simpleKey = ksLookupByName(config, "/simple", KDB_O_NONE);
```

Comment:
proably not the best name for the variable, but the idea was to use the `simple` as a configuration parameter to output the keyset the same way simpleini does (no "real" sections, just section/key = value pairs)